src/main/scala/io/sdkman/changelogs/JavaMigrations.scala

```diff
@@ -237,4 +237,14 @@ class JavaMigrations {
       .validateUrl()
       .insert()
   }
+
+  @ChangeSet(order = "027", id = "027-add_graalvm_1_0_0_rc_3", author = "wololock")
+  def migrate027(implicit db: MongoDatabase) =
```

Comment:
Could you please also remove the old version `1.0.0-rc1-graal` as part of this PR?